xml/dhcp_vendor_class.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="dhcp_vendor_class" protocol="dhcp" database_type="service">
+  <!--
+    Fingerprint definitions that are matched against the string values in the
+    dhcp message vi_vendor_class field
+    The vi_vendor_class field can be found in client discover (1), request (3)
+    and inform (8) messages.
+  -->
+
+  <fingerprint pattern="^Mfg=(?:Fuji)?(?i:Xerox);Typ=(?:MFP|printer);Mod=(?:Xerox )?(\S+) ([a-zA-Z0-9]+)">
+    <description>Xerox Multifunction Printer</description>
+    <example hw.family="VersaLink" hw.model="C405">Mfg=Xerox;Typ=MFP;Mod=VersaLink C405;Ser=</example>
+    <example hw.family="AltaLink" hw.model="C8035">Mfg=Xerox;Typ=MFP;Mod=Xerox AltaLink C8035 Multifunction Printer;Ser=</example>
+    <example hw.family="WorkCentre" hw.model="3345">Mfg=XEROX;Typ=MFP;Mod=WorkCentre 3345;Ser=3BT844742;Loc=</example>
+    <example hw.family="Phaser" hw.model="6500DN">Mfg=FujiXerox;Typ=printer;Mod=Phaser 6500DN;Ser=YXE135427</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="Xerox"/>
+    <param pos="1" name="hw.family"/>
+    <param pos="2" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.family} {hw.model}"/>
```

Comment:
Constructing `hw.product` will work well for these printers since there is so much variation. That being said, it will prevent CPE values for being generated dynamically.  If we see specific models that are seen at high volumes and/or that have high risk vulnerabilities we may wish to construct a specific fingerprint for them with a static product. That will allow CPE values to be generated.